src/matrix/inversewishart.jl

```diff
@@ -110,7 +110,8 @@ end
 function invwishart_logc0(df::Real, Ψ::AbstractPDMat)
     h_df = df / 2
```

Comment:
Since you're already modifying this line, maybe improve `Float32` compatibility here? AFAICT currently `df::Int` and `::AbstractPDMat{Float32}` would result in a `logc0::Float64` due to this line.
```suggestion
```

Reply:
I'm not sure if this is a problem because 
https://github.com/JuliaStats/Distributions.jl/blob/e470421079f2eb4f725657842fbbab40bd981fc8/src/matrix/inversewishart.jl#L40-L43

means `InverseWishart(5, ScalMat(5, 1f0))` will call `InverseWishart(5f0, ScalMat(5, 1f0))`. I've nevertheless fixed it so that `invwishart_logc0(::Int, PDMat{<:Float32})` doesn't return Float64.